Src/FluentAssertions.Core/Core.pcl.csproj

```diff
@@ -30,11 +30,11 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>pdbonly</DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>..\..\Artifacts\Lib\portable-monotouch+monoandroid+xamarin.ios\</OutputPath>
+    <OutputPath>bin\Release\</OutputPath>
```

Comment:
No need because the netstandard 1.3 version is a "fully functional" version and Xamarin supports it. There's no need for a specifically compiled Xamarin version anymore.